mythril/laser/ethereum/transaction/transaction_models.py

```diff
@@ -40,53 +40,74 @@ def __init__(
         self.op_code = op_code
 
 
-class MessageCallTransaction:
-    """ Transaction object models an transaction"""
+class BaseTransaction:
+    """Basic transaction class holding common data."""
 
     def __init__(
         self,
         world_state: WorldState,
-        callee_account: Account,
-        caller: ExprRef,
+        callee_account: Account = None,
+        caller: ExprRef = None,
         call_data=None,
```

Comment:
Not every transaction has calldata (creation transactions)

Reply:
However, I do see that the previous implementation also had this in there

Reply:
Wouldn't it be more correct to make call_data a variable only in the message call transaction?
Although  your solution would probably also result in correct behaviour